PRIVACY.md

```diff
@@ -27,6 +28,8 @@ relationship, performing requests or for billing purposes.
 2.3. We pass your data on to third parties where this is necessary for
 invoicing purposes (e.g. carrying out transactions or posting invoices) or for
 other reasons to enable us to fulfill our contractual obligations to you.
+We inform you about all of the third party services we use and the purpose of
```

Comment:
This sentence was previously in paragraph 11 that has been removed from this version of the privacy policy.